app/assets/javascripts/application.js

```diff
@@ -27,6 +27,22 @@
 
 $(function () {
   $('[data-toggle="tooltip"]').tooltip()
+  
+  if($('#htn-controlled-table').length) {
```

Comment:
Let's pull these table names out into constants and just use them in both the `if` check and the actual element fetch.

Reply:
I think we can pull these selectors out into the specific page they refer to like `reports.js` or `user_analytics.js` etc.

Reply:
Thanks @kitallis, I went ahead and moved all reports-specific selectors to `reports.js`. I'll create a separate story and PR to move analytics table selector to `user_analytics.js`.